bouncer/sentry.py

```diff
@@ -0,0 +1,31 @@
+"""Report exceptions to Sentry/Raven."""
+import raven
+
+
+def raven_tween_factory(handler, _):
+    """Return a tween that reports uncaught exceptions to Sentry."""
+    def raven_tween(request):
+        """Report uncaught exceptions to Sentry."""
+        try:
+            return handler(request)
+        except:
+            request.raven.captureException()
+            raise
+
+    return raven_tween
+
+
+def raven_client(request):
+    """Return the raven.Client object."""
+    request.add_finished_callback(
+        lambda request: request.raven.context.clear(),
+    )
+
+    return raven.Client(
+        dsn=request.registry.settings.get("sentry_dsn"),
+    )
+
+
+def includeme(config):
+    config.add_request_method(raven_client, name="raven", reify=True)
+    config.add_tween("bouncer.sentry.raven_tween_factory")
```

Comment:
Didn't see this comment. I've [tweaked things so that 404s caught by append_slash=True are no longer reported to Sentry](https://github.com/hypothesis/bouncer/commit/fe76bc6204bc0d76d583cf9fe0c788bd5792bd24). 
If we add an exception view that swallows certain kinds of exceptions that don't need reporting - those would still get reported to Sentry.

However if we just wanted to add an exception view to make it more pretty or useful when something goes wrong, we could do that now and the exceptions would still be reported.